security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -24,17 +42,20 @@ The following advisory is available for the {cert-manager-operator} 1.13.0:
 Version `1.13.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.13.3`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.13/#v1133[cert-manager project release notes for v1.13.0].
 
 [id="cert-manager-operator-new-features-1-13"]
+[id="cert-manager-operator-new-features-1-13-0"]
 === New features and enhancements
 
 * You can now manage certificates for API Server and Ingress Controller by using the {cert-manager-operator}.
-For more information, see xref:../../security/cert_manager_operator/cert-manager-creating-certificate.adoc#cert-manager-creating-certificate[Configuring certificates with an issuer].
+For more information, see xref:../../security/cert_manager_operator/cert-manager-creating-certificate#cert-manager-creating-certificate[Configuring certificates with an issuer]. 
 
-* With this release, the scope of the {cert-manager-operator}, which was previously limited to the {product-title} on AMD64 architecture, has now been expanded to include support for managing certificates on {product-title} running on {ibm-z-name} (`s390x`), {ibm-power-name} (`ppc64le`) and ARM64 architectures.
+* With this release, the scope of the cert-manager Operator for Red Hat OpenShift, which was previously limited to the OpenShift Container Platform on AMD64 architecture, has now been expanded to include support for managing certificates on OpenShift Container Platform running on IBM Z® (s390x), IBM Power® (ppc64le) and ARM64 architectures.
```

Comment:
Why is this content changed?

Reply:
While creating the PR with the previous commit, it was deleted, I had to manually add it.

Reply:
I have reverted the changes